torq.q

```diff
@@ -567,27 +553,28 @@ sys:{[cmd]
 	@[{result:system x;.lg.o[`system;"successfully executed"];result};cmd;catcherror[cmd;]]
 	};
 
+// Load additional config files, check if the specific configuration environment variable exists and load the settings if it does
+loadaddconfig:{[envvar;cnfgpath]
+	evstring:string envvar;
+	if[""~getenv[envvar];
+	.lg.o[`fileload;"environment variable ",evstring," not set, not loading this config"];
+	:()];
```

Comment:
```suggestion
	if[""~getenv[envvar];
		.lg.o[`fileload;"environment variable ",evstring," not set, not loading this config"];
		:();
	];
```

just a minor formatting thing - adding indentation here & putting the closing `]` on a new line helps make it immediately clear where the conditional lines end